package.json

```diff
@@ -92,37 +87,33 @@
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
     "@ember/string": "^3.0.1",
-    "@ember/test-helpers": "^2.9.4",
+    "@ember/test-helpers": "3.1.0",
```

Comment:
**Technical risk**

I couldn't install `@ember/test-helpers@3.2.0` because it caused a type error:

```
❯ yarn lint:types                                                                                  
yarn run v1.22.19
$ tsc --noEmit
addon-test-support/setup-intl.ts:87:18 - error TS2551: Property 'unregister' does not exist on type 'Owner'. Did you mean 'register'?

87       this.owner.unregister('service:intl');
                    ~~~~~~~~~~

  node_modules/@types/ember__owner/index.d.ts:114:5
    114     register(fullName: FullName, factory: Factory<object> | object, options?: RegisterOptions): void;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'register' is declared here.
```

Reply:
Maybe related: https://github.com/emberjs/ember-test-helpers/issues/1428